draft-ietf-quic-transport.md

```diff
@@ -712,23 +737,26 @@ sequence of frames, as described in {{frames}}.
 
 ## Connection ID {#connection-id}
 
-QUIC connections are identified by their 64-bit Connection ID.  All long headers
-contain a Connection ID.  Short headers indicate the presence of a Connection ID
-using the Omit Connection ID flag.  When present, the Connection ID is in the
-same location in all packet headers, making it straightforward for middleboxes,
-such as load balancers, to locate and use it.
+A connection ID is used to ensure consistent routing of packets.  The long
+header contains two connection IDs: the Destination Connection ID is chosen by
+the recipient of the packet and is used to provide consistent routing; the
+Source Connection ID is used to set the Destination Connection ID used by a
+peer.
 
-The client MUST choose a random connection ID and use it in Initial packets
-({{packet-initial}}) and 0-RTT packets ({{packet-protected}}).
+During the handshake, packets with the long header are used to establish the
+Destination Connection ID that each endpoint uses.  Each endpoint sends the
+value that they wish their peer to use in the Source Connection ID field.  Upon
+receiving a packet, each endpoint sets the Destination Connection ID to match
+the value of the Source Connection ID that they receive.
```

Comment:
Discussion of receiving multiple packets with differing Source CID fields might be warranted.  Do I set this value every time I receive a packet?  First time I see a change?  What if the change isn't until later?

Reply:
Good catch.  I've put in a rule for this.  I think that it's a sensible rule, but you can judge that for yourself.